src/core/api.ts

```diff
@@ -88,10 +88,10 @@ class API {
    * @param args An array of arguments to pass as Textile args headers
    * @param opts An object of options to pass as Textile options headers
    */
-  protected async sendGatewayGet(path: string, args?: string[], opts?: KeyValue, headers?: KeyValue) {
```

Comment:
Simplifying the signature, because the gateway doesn't support these args and headers